spec/setup_test_database.rb

```diff
@@ -0,0 +1,13 @@
+require 'pg'
+
+def set_up_test_database!
+
+    p "Setting up test database..."
+
+    con = PG.connect :dbname => 'chitter_test'
+
+    con.exec("TRUNCATE chitter_table;")
+
+    con.exec("INSERT INTO chitter_table(user_id, peep) VALUES ('test_user', 'Peep peep!');")
+
+end
```

Comment:
clean test set up instructions, might want to look into Truncate vs Drop table for set up.